napari/_qt/layer_controls/qt_image_controls.py

```diff
@@ -114,6 +118,11 @@ def __init__(self, layer):
         self.grid_layout.setColumnStretch(1, 1)
         self.grid_layout.setSpacing(4)
 
+        # Similarly, the colormap combo box will be hidden if the layer is rgb
+        if layer.rgb:
+            self.colorbarLabel.hide()
```

Comment:
If you see my other comment about leaving the adding of the widgets and creating `self.colormapLabel`, then here we just need to add `self.colormapLabel.hide()` and everything should work fine